src/main.js

```diff
@@ -343,7 +343,7 @@ client.on("message", function (topic, payload) {
     console.log(
       isOwnPayload(contents) ? "🦝" : "🤖",
       topic,
-      contents.substring(0, 100),
+      JSON.stringify(contents).substring(0, 100),
```

Comment:
No need to stringify again if we have the original payload.
```suggestion
      payload.substring(0, 100),
```